README.md

```diff
@@ -22,7 +20,210 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+### Exporting
+
+The `LiveFixtures::Export` module is meant to be included into your export class.
+
+
+    class Export::User
+      include LiveFixtures::Export
+
+      def export(user_id)
+        set_export_dir "#{Rails.root}/data/export/user/#{user_id}/"
+
+        export_user_and_posts(user_id)
+      end
+
+      def export_user_and_posts(user_id)
+        user = User.find(user_id)
+        export_fixtures([user])
+
+        export_fixtures user.posts, :user do |post|
+          { "likes" => post.likes.count,
+            "unique_url" => Template.new("<%= Export::Helper.unique_url %>") }
+        end
+      end
+    end
+
+
+1. Call #set_export_dir to set the dir where files should be created.
+   If the dir does not already exist, it will be created for you.
+
+2. Then call #export_fixtures for each db table, which will produce
+   one yml file for each db table. Do *not* call export_fixtures multiple
+   times for the same db table - that will overwrite the file each time!
+
+3. For advanced usage, read the sections about Additional Attributes, References, and Templates.
+
+### Importing
+
+The `LiveFixtures::Import` class allows you to specify the location of your fixtures and the order in which to import them. Once you've done that, you can import them directly to your database.
+
+
+    module Seed::User
+      def self.from_fixtures(fixtures_directory)
+        insert_order = %w{users posts}
+
+        importer = LiveFixtures::Import.new fixtures_directory, insert_order
+        importer.import_all
+      end
+    end
+
+### Advanced Usage
```

Comment:
Maybe this should be a `##`? I got confused at first thinking `Advanced Usage` was a self-contained example.